components/languages/README.md

```diff
@@ -11,13 +11,39 @@ Rows in the language table can then be referenced from other tables using a
 
 ## What's a language?
 
-For the purposes of CLDF, a language is one of the objects whose attributes are studied, and stored
-as values for [parameters](../parameters). Thus, if a dataset contains the results of a sociolinguistic
-study, rows in `LanguageTable` will be likely be sociolects. Whether and how such rows are related to
-languoids as understood for example by Glottolog can be indicated in the [Glottocode](http://cldf.clld.org/v1.0/terms.rdf#glottocode) column.
+For the purposes of CLDF, a language is one of the objects whose attributes are described in a dataset.
```

Comment:
"For the purposes of CLDF, a language is one of the objects described in a dataset, along with its attributes."